### PR TITLE
GeoHack 2020 to demo site

### PR DESCRIPTION
**Pull request by AtherBilal**
solves: #200 
Adding the 2020 winners code to the demo site

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      "email": "bather@1904labs.com",
+      "url": "https://github.com/AtherBilal"
+    }
+  ],
+  "homepage": "https://ol-kit.com",
+  "bugs": "https://github.com/MonsantoCo/ol-kit/issues",
+  "main": "./core/index.js",
+  "scripts": {
+    "app:build": "webpack --config webpack.config.js",
+    "app": "NODE_ENV=development webpack-dev-server --hot --inline --config webpack.config.js",
```
Comment (AtherBilal) [inline app/package.json:46]: remove NODE_ENV file, change to `OL_KIT_DEVELOPMENT = true`

Comment (AtherBilal) [reply app/package.json:46]: in root `package.json`

File: package.json [hunk 2]
```diff
@@ -82,6 +87,7 @@
     "react-hook-form": "~5.7.2",
     "react-select": "~3.1.0",
     "shp-write": "~0.3.2",
+    "ugh": "^0.1.1",
```
Comment (stazrad) [inline package.json:90]: yooo ugh is an internal util (see `~/src/ugh.js`)

Comment (AtherBilal) [reply package.json:90]: kk removing

Comment (stazrad) [reply package.json:90]: @AtherBilal hey this one is still in there

**Review by stazrad — APPROVED** [review]

**Pull request merged**